stryker.conf.json

```
@@ -7,15 +7,12 @@
     "progress",
     "dashboard"
   ],
-  "testRunner": "command",
+  "testRunner": "jest",
+  "coverageAnalysis": "perTest",
```

I would like to try changing `coverageAnalysis` back:

```suggestion
  "coverageAnalysis": "off",
```

When I tried this, it seemed to be testing 2014 mutations again.

---

I would have thought/expected, that this would be the most important part of speeding up the mutation testing, since it only needs to execute those tests for any mutation, that would be affected by it.

---

Sounds right *in theory*. When I tried `off` vs `perTest` vs `all`, both `perTest` and `all` resulted in 1800+ mutations while `off` resulted in 2014 mutations (same as command runner). I recall that the initial focus of Stryker 4 was the mutation switching and that improving the Jest runner would be done sometime as a followup.

My primary purpose of this proposal was to be ready for the future, assuming that they will make some improvements in the Jest runner.